src/routes/pages/CartPage/CartPage.tsx

```diff
@@ -0,0 +1,83 @@
+import '../../../App.css';
+
+import Header from '../../../components/common/Header/Header';
+import HeaderButton from '../../../components/common/Header/HeaderButton';
+import ContainerLayout from '../../../components/common/ContainerLayout/ContainerLayout';
+import CartListTitle from '../../../components/CartListTitle/CartListTitle';
+import CartItem from '../../../components/CartItem/CartItem';
+import CartList from '../../../components/CartList/CartList';
+import CartPriceInfo from '../../../components/CartPriceInfo/CartPriceInfo';
+import OrderButton from '../../../components/OrderButton/OrderButton';
+import EmptyCart from '../../../components/EmptyCart/EmptyCart';
+
+import { Logo } from '../../../assets';
+import { cartPrice } from '../../../utils/cartPrice/cartPrice';
+
+import useCartList from '../../../hooks/useCartList';
+import useSelect from '../../../hooks/useSelect';
+import LoadingSpinner from '../../../components/common/LoadingSpinner/LoadingSpinning';
+
+function CartPage() {
+  const {
+    cartList,
+    isLoading,
+    increaseCartItem,
+    decreaseCartItem,
+    deleteCartItem,
+  } = useCartList();
+
+  const {
+    selectedItems,
+    isAllSelected,
+    handleSelectItem,
+    handleSelectAllItems,
+  } = useSelect(cartList);
+
+  const totalPrice = cartPrice.totalPrice(cartList, selectedItems);
+
+  const renderCartList = () => {
+    return cartList.length === 0 ? (
+      <EmptyCart />
+    ) : (
+      <>
+        <CartList
+          isAllSelected={isAllSelected}
+          handleSelectedAllItems={handleSelectAllItems}
+        >
+          {cartList.map((cartItem) => (
+            <CartItem
+              key={cartItem.id}
+              cartItem={cartItem}
+              isSelected={selectedItems.includes(cartItem.id)}
+              handleSelectItem={handleSelectItem}
+              increaseCartItem={increaseCartItem}
+              decreaseCartItem={decreaseCartItem}
+              deleteCartItem={deleteCartItem}
+            />
+          ))}
+        </CartList>
+        <CartPriceInfo totalPrice={totalPrice} />
```

Comment:
마찬가지로 이런 모습으로 쓰이면 어떨까 싶어요!

```suggestion
        <CartPriceInfo value={totalPrice} />
```

혹은

```suggestion
        <CartPriceInfo>{totalPrice}</CartPriceInfo>
```

Reply:
저는 개인적으로 totalPrice로 전달해주는 게 자식 컴포넌트를 보았을 때도 직관적이지 않을까 생각해서 네이밍을 동일하게 작성해보았습니다! 혹시 해먼드가 두 방식을 제안해주신 이유가 궁금합니다!ㅎㅎ

Reply:
컴포넌트의 이름이 CartPrice 라서, 어차피 CartPrice 에 넘겨주는 값은 price 이기 때문에 value 혹은 data 처럼 표현하면 좋지 않을까? 라고 생각했어요!